lib/commands/context.js

```diff
@@ -109,9 +108,8 @@ commands.getWindowHandles = async function () {
     throw new errors.NotImplementedError();
   }
 
-  let pageArray = await this.listWebFrames();
-  this.windowHandleCache = _.map(pageArray, this.massagePage);
```

Comment:
perhaps it should be `this.mEssagePage`?

Reply:
I don't understand.

Reply:
I mean the property name is misspelled

Reply:
Oh. No. `massagePane` used to exist. It was never ported over when we rewrote.